### DISCUSSION TITLE
Alter session param is not persisting

### DISCUSSION THREAD

**Discussion by puneetloya** [sijms/go-ora#570] [Q&A]
I am using this project: https://github.com/burningalchemist/sql_exporter, added Oracle as a driver to it.  I am able to successfully query to the Oracle database. My setup is like:
* I have multiple oracle driver objects(connections) in the same golang code, querying multiple databases.
* Each connection is setup optionally with Alter session parameters as query datetimes are in a certain timezone(not SYSDATE :( ).
* Note I am explicitly calling Alter session using: 
```
go_ora.AddSessionParam("TIME_ZONE", "Europe/London")
```
This param is working intermittently and I am not sure why.

**Comment by sijms** [comment]
If you have multiple oracle connection you should use 
[connect to multiple database](https://github.com/sijms/go-ora/tree/v2.7.24?tab=readme-ov-file#connect-to-multiple-database)
then pass each db object to `go_ora.AddSessionParam`
